src/hooks/member.ts

```diff
@@ -357,6 +358,7 @@ export const useMemberNotesAdmin = (
             ]
           : undefined,
     },
+    description: { _like: `%${keyword}%` },
```

Comment:
FIX: if there no keyword, don't need description condition
because empty like condition can not fetch null description